scripts/table_updates/README.md

```diff
@@ -6,11 +6,14 @@ BPC Table Update
 Installation and Setup
 ----------------------
 ### Python version
-Make sure you have Python 3.8 installed
+Make sure you have Python 3.8 installed.
```

Comment:
I think we should update this to be 3.9 and above to be compatible with Python client.

Reply:
Ill let you update your PR first, then Ill make some changes to this PR.